app/controllers/api_completed_learning_units_controller.rb

```diff
@@ -7,4 +7,16 @@ def curriculums_learning_units_completed_by_user
     )
     render json: completed_learning_units, only: %i[id learning_unit_id]
   end
+
+  def uncomplete_learning_unit
+    learning_unit_id = LearningUnit.find(params[:learning_unit_id]).id
+    if CompletedLearningUnit.delete_by(
+      user: current_user,
+      learning_unit_id:
+    ).positive?
+      render_deletion_message
+    else
+      handle_record_invalid_error
```

Comment:
according to this thread https://stackoverflow.com/questions/6439416/status-code-when-deleting-a-resource-using-http-delete-for-the-second-time
I would use 404 for subsequent delete requests so its clear that the record does not exist anymore and it's not found. 400 is not clear if it exists or not after the submission of the request

Reply:
Done